src/components/Treeview/Label.js

```diff
@@ -1,7 +1,7 @@
 import styled from "styled-components";
 import Icon from "../Icon";
 import { getThemeProp, ifFlag } from "../../utils";
-import { branchHeight } from "./settings";
+import { branchLength } from "./settings";
```

Comment:
Why make this change? Why not instead set the default value (and value in theme) to a different number?

Reply:
All right, this checks out.